src/node_crypto.cc

```diff
@@ -5220,6 +5222,16 @@ bool PublicKeyCipher::Cipher(Environment* env,
       return false;
   }
 
+  if (oaep_label_len != 0) {
+    // OpenSSL takes ownership of the label, so we need to create a copy.
+    void* label = OPENSSL_memdup(oaep_label, oaep_label_len);
+    CHECK_NOT_NULL(label);
+    if (!EVP_PKEY_CTX_set0_rsa_oaep_label(ctx.get(), label, oaep_label_len)) {
```

Comment:
I think `EVP_PKEY_CTX_set0_rsa_oaep_label()` can only fail when the padding mode isn't `RSA_PKCS1_OAEP_PADDING`? Maybe turn it into a CHECK_GT if that's the case.

At any rate, I believe the return value check should be that it's > 0 to determine it was successful.